testing/upload_validation.ts

```diff
@@ -1682,6 +1682,14 @@ const legacyPackMetadataSchema = validateFormulas(
 
     (data.syncTables || []).forEach((syncTable, i) => {
       const schema: ObjectSchema<any, any> = syncTable.schema;
+      const propertyNames = Object.keys(schema.properties);
+      for (const dupe of getNonUniqueElements(propertyNames)) {
+        context.addIssue({
+          code: z.ZodIssueCode.custom,
+          path: ['syncTables', i, 'properties', dupe],
+          message: `Sync table schema property names must be unique. Found duplicate name "${dupe}" in sync table ${syncTable.name}.`,
```

Comment:
This is doing a case-insensitive uniqueness check, so the message should probably mention that, otherwise the maker will probably be confused because the values are indeed literally unique.